src/components/ProgramRecordAlert/ProgramRecordAlert.jsx

```diff
@@ -59,23 +60,21 @@ const ProgramRecordAlert = ({
             </Alert.Heading>
             <FormattedMessage
               id="send.failure.alert.message"
-              defaultMessage="You can try to send your record to the {pathway_name} pathway again. If this issue persists"
+              defaultMessage="You can try to send your record to the {pathway_name} pathway again. If this issue persists {link}"
               description="Message for the alert that displays when the program record fails to send"
               values={{
                 pathway_name: creditPathway.name,
+                link: (
+                  <Hyperlink
+                    destination={`${getConfig().SUPPORT_URL_LEARNER_RECORDS}`}
+                    target="_blank"
+                    showLaunchIcon={false}
+                  >
+                    {`contact ${platform} support.`}
```

Comment:
This string still looks fishy to me too. I fear this wouldn't get translated correctly.